### PR TITLE
Add some configuration files and more scripts

### PR DESCRIPTION
**Pull request by colinperepelken**
- Add `tsconfig.json`
- Add `cspell.json`
- Add some scripts to `package.json` and update `README`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,8 +4,10 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "npx ts-node src/main.ts",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "node ./dist/main.js",
+    "dev": "npx ts-node src/main.ts",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "tsc"
```
Comment (mackosx) [inline package.json:10]: **praise**: This is a helpful addition, thanks.

File: cspell.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "version": "0.2",
+  "language": "en",
+  "words": ["neighbour", "neighbours", "pathing"]
```
Comment (mackosx) [inline cspell.json:4]: **question**: Whats this file for?

File: README.md [hunk 3]
```diff
@@ -2,13 +2,41 @@
...
+```sh
+npm run build
+npm start
+```
+
+## `TODO`
 
 - [x] Follow the set up tutorial
+- [x] Add `tsconfig.json` with strict type checking
+- [ ] Add `eslint` and `prettier`
```
Comment (mackosx) [inline README.md:39]: 👍🏻

File: package.json [hunk 4]
```diff
@@ -4,8 +4,10 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "npx ts-node src/main.ts",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "node ./dist/main.js",
+    "dev": "ts-node src/main.ts",
```
Comment (colinperepelken) [inline package.json:8]: Removed `npx` as you should be able to run this with just `ts-node` if you install it.

Comment (mackosx) [reply package.json:8]: I have no clue, if it works locally good enough for me.

**Review by mackosx — APPROVED** [review]
Lovely, LGTM

File: package.json (hunk 4 above)
Comment (mackosx) [reply package.json:8]: Had to look up what `npx` is, seems like it should be fine since its in dev deps.

File: cspell.json (hunk 2 above)
Comment (colinperepelken) [reply cspell.json:4]: Otherwise VS Code complains about spelling errors. `words` is a list of words to ignore the spelling for.

**Pull request merged**